mercure.rst

```diff
@@ -645,13 +645,14 @@ During functional testing you can instead decorate the Hub::
     }
 
 HubStub decorates the default hub service so no updates are actually
-sent. Here is the HubStub implementation::
+sent. Here is the HubStub implementation:
+
+.. code-block:: yaml
```

Comment:
Should I revert this and instead focus on https://github.com/symfony-docs-tools/code-block-checker/issues/20

There are many more yaml/json/xml code blocks like this.

Reply:
This is a bug, so please fix them :) (and not the checker)